WordPress/Classes/ViewRelated/Blog/Blog Dashboard/ViewModel/BlogDashboardViewModel.swift

```diff
@@ -150,12 +157,14 @@ private extension BlogDashboardViewModel {
         let scrollView = viewController?.mySiteScrollView
         let position = scrollView?.contentOffset
 
+        let isScrollNeeded = !isReloadingPersonalizationSettings
         dataSource?.apply(snapshot, animatingDifferences: false) { [weak self] in
             guard let scrollView = scrollView, let position = position else {
                 return
             }
-
-            self?.scroll(scrollView, to: position)
+            if isScrollNeeded {
+                self?.scroll(scrollView, to: position)
```

Comment:
I'm not sure if the `scroll(_:to:)` code is needed. It was added in the [following MR](https://github.com/wordpress-mobile/WordPress-iOS/pull/18245). I tried removing it, and I can't reproduce the defect described in the attached issue. 

I prefer to remove and simplify things, but I'm not sure I tested it correctly.